packages/@webex/webex-core/test/unit/spec/credentials/scope.js

```diff
@@ -0,0 +1,46 @@
+import {assert} from '@webex/test-helper-chai';
+import {sortScope, filterScope, diffScopes} from '@webex/webex-core/src/lib/credentials/scope';
+
+describe('webex-core', () => {
+  describe('scope utils', () => {
+    describe('sortScope', () => {
+      it('should sort scopes alphabetically', () => {
+        assert.equal(sortScope(undefined), '');
+        assert.equal(sortScope(''), '');
+        assert.equal(sortScope('a'), 'a');
+        assert.equal(sortScope('b c a'), 'a b c');
+      });
+    });
+
+    describe('filterScope', () => {
+      it('should filter out one filter from the filter list and sort the result', () => {
+        assert.equal(filterScope('a', undefined), '');
+        assert.equal(filterScope('a', ''), '');
+        assert.equal(filterScope('a', 'a'), '');
+        assert.equal(filterScope('a', 'a b c'), 'b c');
+        assert.equal(filterScope('c', 'a c b'), 'a b');
+      });
+    });
+
+    describe('diffScopes', () => {
+      it('should return empty string when the first scope is subset of the second one', () => {
```

Comment:
suggestion: 'should return an empty string, if all items in the first scope are contained in the second scope'
I know your description is accurate, but it requires more mental effort to interpret I think